src/MonitorProfiler/Logging/LogLevelHelper.h

```diff
@@ -16,35 +16,36 @@ class LogLevelHelper final
     /// <summary>
     /// Gets the short name of the log level.
     /// </summary>
-    static HRESULT GetShortName(LogLevel level, tstring& strName)
+    static HRESULT GetShortName(LogLevel level, lstring& strName)
     {
         // The log levels are intentionally four characters long
         // to allow for easy horizontal alignment.
         switch (level)
         {
         case LogLevel::Critical:
-            strName.assign(_T("crit"));
+            strName.assign(_LS("crit"));
             return S_OK;
         case LogLevel::Debug:
-            strName.assign(_T("dbug"));
+            strName.assign(_LS("dbug"));
             return S_OK;
         case LogLevel::Error:
-            strName.assign(_T("fail"));
+            strName.assign(_LS("fail"));
             return S_OK;
         case LogLevel::Information:
-            strName.assign(_T("info"));
+            strName.assign(_LS("info"));
             return S_OK;
         case LogLevel::None:
-            strName.assign(_T("none"));
+            strName.assign(_LS("none"));
             return S_OK;
         case LogLevel::Trace:
-            strName.assign(_T("trce"));
+            strName.assign(_LS("trce"));
             return S_OK;
         case LogLevel::Warning:
-            strName.assign(_T("warn"));
+            strName.assign(_LS("warn"));
             return S_OK;
         default:
-            return E_FAIL;
+            strName.assign(_LS("ukwn"));
+            return S_OK;
```

Comment:
It looks like this method will return `S_OK` no matter what - is this intentional, and if so, do we even need to return `HRESULT` and use `IfFailRet` on it?

Reply:
This was intentional. I didn't want to have to rewrite the callers.